src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,108 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exseption.InvalidAgeException;
+import core.basesyntax.exseption.InvalidLoginException;
+import core.basesyntax.exseption.InvalidPasswordException;
+import core.basesyntax.exseption.LoginIsTakenException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static String VALID_LOGIN_1;
+    private static String VALID_LOGIN_2;
+    private static String VALID_PASSWORD;
+    private static String INVALID_PASSWORD;
+    private static Integer AGE_OVER_18;
+    private static Integer AGE_UNDER_18;
+    private static Integer AGE_18;
```

Comment:
These fields should make constant ```private static final``` and you can initialize it here.  Also, it is bad practice to create variable names with counters or numbers (good example: INVALID_AGE, VALID_LOGIN etc).